README.md

```diff
@@ -18,6 +19,18 @@ make toolchain
 make build
 ```
 
+Optional variables:
+
+```
+CORE_BPF_PROGRAM_ID=... # see below
+CORE_BPF_TARGET=...     # see below
+```
+
+When the `CORE_BPF_PROGRAM_ID` environment variable is set, SolFuzz-Agave will
+overwrite the specified builtin program in the program cache with the provided
+BPF target. Provide the path to the compiled BPF program target `.so` file with
+variable `CORE_BPF_TARGET`.
+
```

Comment:
It might be nice to also log a message in `sol_compat_instr_execute_v1`, something like:
- If these variables are set: "Replacing $(CORE_BPF_PROGRAM_ID) with $(CORE_BPF_TARGET)"
- If not, "Using default core BPF programs"

Reply:
I can add this log, sure!

Reply:
Are you sure we want logs in the fuzz instruction? There doesn't seem to be any others at the moment, and that's going to get invoked a lot of times.